test_conformance/extensions/cl_khr_command_buffer/basic_command_buffer.cpp

```diff
@@ -546,4 +546,4 @@ int test_out_of_order(cl_device_id device, cl_context context,
                       cl_command_queue queue, int num_elements)
 {
     return MakeAndRunTest<OutOfOrderTest>(device, context, queue, num_elements);
-}
+}
```

Comment:
The new files you've added in this PR end in a new line, so think we should keep this new line as-is